test/boostTest.cpp

```diff
@@ -144,7 +144,7 @@ test_suite* init_unit_test_suite( int /*argc*/, char* /*argv*/[] )
 			master,
 			options.testPath / ts.path,
 			ts.subpath,
-			options.ipcPath,
+			options.ipcPath.c_str(),
```

Comment:
Similar question - why not keep the type?

Reply:
If by "keep" you mean "change it so it's the same everywhere":
I would need to change the existing type `std::string` to `boost::filesystem::path` in at least 3+ different classes. 
When I was investigating that it seemed not worth the effort. I can check it out again

Reply:
Ah, I didn't realize it is path somewhere and string somewhere else. Please just use `string()` instead of `c_str()`.